CONTRIBUTING.md

```diff
@@ -31,3 +31,23 @@ Some things that will increase the chance that your pull request is accepted:
 * Use Rails idioms and helpers
 * Include tests that fail without your code, and pass with it
 * Update the documentation, the surrounding one, examples elsewhere, guides, whatever is affected by your contribution
+
+## Getting started
+
+To setup the app, you'll want to do the following:
+
+1. Clone the repo:
+  - SSH: `git clone git@github.com:rubyforgood/coral.git`
+  - HTTPS `git clone https://github.com/rubyforgood/coral.git`
+2. Change to the app directory: `cd coral`
+3. Be sure Postgres is [installed](https://www.postgresql.org/download/) (version 9.1+) and running, or that you've started it locally [in a container](https://hub.docker.com/_/postgres)
+4. If you're using `rvm` or `rbenv` (you should), ensure you've got the right ruby version selected:
+  - RVM: `rvm current` to check version, `rvm use` to change version
```

Comment:
I am way past the point where I think it's at all acceptable to promote RVM.  It's a bad system, and I don't want to  push it on newbies.

rbenv or chruby  and generally we just need to handle the `.ruby-version` ... which is in the repo; the only check is to make sure that (at current) 2.6.3 is installed.

Reply:
Some of us have it already (for years in my case), so I thought it best to just include that. Not promoting RVM necessarily, but happy to add chruby too.

Reply:
If you've already got RVM, you know how to use it.

Granting it pride of place _is_ advertising it